### ISSUE TITLE
FR: Say which build stage failed in Github PR status

### ISSUE THREAD

**Issue by bradleyjkemp** [travis-ci/travis-ci#8171]
Currently the Github status reported has the description fixed as "The Travis CI build failed" it would be nice to be able to quickly see which build stage caused the failure by putting this in the description (i.e. include it in the message here: https://github.com/travis-ci/travis-tasks/blob/master/lib/travis/addons/github_status/task.rb#L21)

This would let you have a Tests build stage and a Lint build stage and as a reviewer quickly see that even though a build is failing for a PR it's still worth reviewing because it's only failing at the Lint stage

**Comment by Kamilius** [comment]
Agree. It would be awesome to have an ability to send separate GitHub statuses for each available stage.
